src/ert/gui/main.py

```diff
@@ -12,7 +12,7 @@
 from ert.cli.main import ErtTimeoutError
 from ert.gui.ertnotifier import ErtNotifier
 from ert.gui.ertwidgets import SummaryPanel, resourceIcon
-from ert.gui.main_window import GertMainWindow
```

Comment:
Separate issue, but I think prefixing the class name with the app name is generally not a good idea. so some other renaming that could be done is `ErtNotifier` to `Notifier`, `ErtTimeoutError` to `TimeoutError`, `ErtMainWindow` to `MainWindow` etc.

Reply:
```
$ grep -r -e "class Ert[A-Z]" --include=*.py
tests/unit_tests/storage/test_extraction.py:class ErtConfigBuilder:
ecl/python/ecl/util/test/ert_test_context.py:class ErtTest(BaseCClass):
ecl/python/ecl/util/test/ert_test_context.py:class ErtTestContext(object):
ecl/python/ecl/util/test/ert_test_runner.py:class ErtTestRunner(object):
src/ert/exceptions/_exceptions.py:class ErtError(Exception):
src/ert/analysis/_es_update.py:class ErtAnalysisError(Exception):
src/ert/cli/main.py:class ErtCliError(Exception):
src/ert/cli/main.py:class ErtTimeoutError(Exception):
src/ert/shared/models/base_run_model.py:class ErtRunError(Exception):
src/ert/shared/plugins/plugin_manager.py:class ErtPluginManager(pluggy.PluginManager):
src/ert/shared/plugins/plugin_manager.py:class ErtPluginContext:
src/ert/shared/_doc_utils/ert_jobs.py:class ErtForwardModelDocumentation(_ErtDocumentation):
src/ert/shared/_doc_utils/ert_jobs.py:class ErtWorkflowDocumentation(_ErtDocumentation):
src/ert/_c_wrappers/enkf/enums/ert_impl_type_enum.py:class ErtImplType(BaseCEnum):
src/ert/_c_wrappers/enkf/ert_workflow_list.py:class ErtWorkflowList:
src/ert/_c_wrappers/job_queue/ert_plugin.py:class ErtPlugin(ErtScript):
src/ert/_c_wrappers/job_queue/ert_script.py:class ErtScript:
src/ert/gui/ertnotifier.py:class ErtNotifier(QObject):
src/ert/gui/ertwidgets/models/ertsummary.py:class ErtSummary:
src/ert/gui/ertwidgets/message_box.py:class ErtMessageBox(QDialog):
src/ert/gui/main_window.py:class ErtMainWindow(QMainWindow):
```

Maybe not all of them? Anyway, a separate issue ..